package/package.json

```diff
@@ -94,64 +94,64 @@
-    "@op-engineering/op-sqlite": "^9.3.0",
+    "@babel/plugin-transform-arrow-functions": "^7.25.9",
+    "@babel/plugin-transform-shorthand-properties": "^7.25.9",
+    "@babel/plugin-transform-template-literals": "^7.26.8",
+    "@babel/preset-env": "7.26.9",
+    "@babel/preset-typescript": "7.26.0",
+    "@babel/runtime": "^7.26.9",
+    "@op-engineering/op-sqlite": "^11.4.8",
     "@react-native-community/eslint-config": "3.2.0",
     "@react-native-community/eslint-plugin": "1.3.0",
-    "@react-native-community/netinfo": "^11.3.2",
-    "@react-native/babel-preset": "0.73.21",
+    "@react-native-community/netinfo": "^11.4.1",
+    "@react-native/babel-preset": "0.78.0",
     "@testing-library/jest-native": "^5.4.3",
-    "@testing-library/react-native": "12.4.5",
-    "@types/better-sqlite3": "^7.6.0",
-    "@types/eslint": "7.2.10",
-    "@types/jest": "^29.5.12",
-    "@types/linkify-it": "3.0.2",
-    "@types/lodash": "4.14.169",
-    "@types/mime-types": "2.1.0",
-    "@types/react": "18.2.6",
-    "@types/react-test-renderer": "18.0.7",
+    "@testing-library/react-native": "13.1.0",
+    "@types/better-sqlite3": "^7.6.12",
+    "@types/eslint": "9.6.1",
+    "@types/jest": "^29.5.14",
+    "@types/linkify-it": "5.0.0",
+    "@types/lodash": "4.17.16",
+    "@types/mime-types": "2.1.4",
+    "@types/react": "^19.0.0",
+    "@types/react-test-renderer": "19.0.0",
     "@types/use-sync-external-store": "^0.0.6",
-    "@types/uuid": "^8.3.4",
+    "@types/uuid": "^10.0.0",
     "babel-eslint": "10.1.0",
-    "babel-jest": "29.6.3",
-    "babel-loader": "8.2.2",
-    "babel-plugin-module-resolver": "4.1.0",
+    "babel-jest": "29.7.0",
+    "babel-loader": "10.0.0",
+    "babel-plugin-module-resolver": "5.0.2",
     "better-sqlite3": "11.8.1",
     "eslint": "^9.20.1",
-    "eslint-config-prettier": "10.0.1",
+    "eslint-config-prettier": "10.0.2",
+    "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-jest": "^28.11.0",
     "eslint-plugin-markdown": "5.1.0",
     "eslint-plugin-prettier": "5.2.3",
     "eslint-plugin-react": "^7.37.4",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-react-hooks": "^5.1.0",
+    "eslint-plugin-react-hooks": "^5.2.0",
     "eslint-plugin-react-native": "^5.0.0",
-    "i18next-parser": "^9.0.0",
+    "i18next-parser": "^9.3.0",
     "jest": "^29.7.0",
-    "moment-timezone": "^0.5.45",
-    "prettier": "^3.5.2",
-    "react": "18.2.0",
-    "react-native": "0.73.6",
-    "react-native-builder-bob": "0.23.1",
-    "react-native-gesture-handler": "~2.16.1",
-    "react-native-reanimated": "~3.10.0",
-    "react-native-svg": "15.1.0",
-    "react-test-renderer": "18.2.0",
-    "typescript": "5.0.4",
-    "typescript-eslint": "^8.24.1",
-    "uuid": "^8.3.2"
+    "moment-timezone": "^0.5.47",
+    "prettier": "^3.5.3",
+    "react": "19.0.0",
+    "react-native": "0.78.0",
+    "react-native-builder-bob": "0.37.0",
```

Comment:
On video, we ran into some compatibility issues with Expo once we upgraded `react-native-builder-bob` to its latest version.
For some reason, the CJS bundle was trying to load ESM code and caused the expo build to fail. We had to downgrade. Please check if this works in Chat. @santhoshvai knows more about this.

Reply:
Just writing a note here as we discussed this over Slack, there are no issues and it's been tested. If any do come up they won't be obvious ones.

Thanks for pointing it out though !